### DISCUSSION TITLE
Video thumbnail generation

### DISCUSSION THREAD

**Discussion by regul8or** [gerbera/gerbera#2781] [Q&A]
I noticed that thumbnails generated for this folder are completely unrelated

![1](https://user-images.githubusercontent.com/21054394/210386658-cc941241-2e6d-4540-98f5-e6c79017ce63.jpg)

After some searching I found the source for many of them

![2](https://user-images.githubusercontent.com/21054394/210386828-214819e0-57e2-471a-9ca9-6feaa7e74fb3.jpg)

Do you have an idea how this could be?

**Comment by whyman** [comment]
That is strange!

Can you open an issue and re-import those files with debug logging enabled (--debug) so we can see what's happening?

**Reply by regul8or** [reply]
Re-import? Turn off autoscan and then on again?

I noticed that the other device (VLC on Android tablet) displays correct thumbs. Is it possible?
